src/moveit_opw_kinematics_plugin.cpp

```diff
@@ -292,6 +323,13 @@ bool MoveItOPWKinematicsPlugin::searchPositionIK(const std::vector<geometry_msgs
     return false;
   }
 
+  //for all solutions, check if solution +-360° is still inside limits
+  // opw solution might be outside joint limits, extended one inside (asymmetric limits)
```

Comment:
Suggestion: `An opw solution might be outside the joint limits, while the extended one is inside (e.g. asymmetric limits).`
I had to think about the sentence for a while before getting it, maybe making it more of a complete sentence can help future readers?

Reply:
```suggestion
  // An opw solution might be outside the joint limits, while the extended one is inside (e.g. asymmetric limits)
```

Reply:
Hmm, seems I cannot commit my own suggestion :disappointed: in here